src/ctap/storage.rs

```diff
@@ -105,7 +105,7 @@ enum Key {
     MinPinLengthRpIds,
 }
 
-pub struct MasterKeys(Vec<u8>);
+pub struct MasterKeys([u8; 64]);
```

Comment:
If `MasterKeys` is already it's own struct, I'm wondering why the two arrays need to be stored in a single array and have getter functions, compared to how it was before with 2 public arrays you could just use?

Reply:
Oh good point, I'll change that back.